modules/@apostrophecms/modal/ui/apos/mixins/AposDocsManagerMixin.js

```diff
@@ -121,17 +121,16 @@ export default {
     // It would have been nice for this to be computed, however
     // AposMediaManagerDisplay does not re-render when it is
     // a computed prop rather than a method call in the template.
-    maxReached() {
+    maxReached(checkedCount = this.checked.length) {
```

Comment:
Mind blown — seems valid — I see examples like x = getRandomValue() with notes to the effect that yes, it's different every time. Cool